index.rst

```diff
@@ -21,302 +21,434 @@ Some documentation on this system can be accessed on `Docushare Collection-11943
+    .. figure:: /_static/mounted_exhaust.png
+       :name: mounted_exhaust
+       :target: ../_images/mounted_exhaust.png
+       :alt: mounted_exhaust
+       :scale: 10 %
+
+    .. figure:: /_static/exhaust_tube.png
+       :name: exhaust_tube
+       :target: ../_images/exhaust_tube.png
+       :alt: exhaust_tube
+       :scale: 10 %
+
+
+The WLS is controlled using the **ts_ATWhiteLight** CSC. 
+See the `XML documentation <https://ts-xml.lsst.io/sal_interfaces/ATWhiteLight.html>`__ for more information.
+
+Further documentation on the White Light Source can be found on Docushare `here <https://docushare.lsst.org/docushare/dsweb/View/Collection-5178>`__
 
 Chiller
----------
-The chiller is used to cool the water cell in the white light source. This uses a mixture of ethanol and water (50/50)?? to carry heat from a shell surrounding the main water cell. The ethanol helps the water in these tubes from freezing. It does not actually pump the water in and out of the cell, but rather carries heat from tubes surrounding the water cell. The control is on the water coming back to the chiller so there is likely to be some delay in large temperature swings. The temperature should be set to 25C (TBC).
+-------
+The chiller is used to cool the water cell in the white light source. 
+This uses a mixture of methanol and water (10/90) to carry heat from a shell surrounding the main water cell. 
+The water should be distilled water, which can obtained from the DI plant that supports the coating chamber.
+The methonol helps the water in these tubes from freezing. 
+The chiller does not actually pump the water in and out of the cell, but rather carries heat from tubes surrounding the water cell. 
+The control is on the water returning to the chiller so there is likely to be some delay for large temperature swings. 
+The temperature should be set to 25C.
 
-The chiller is controlled via connected to a serial device server (RS-232 to ethernet). The serial device server must run at 9600 baud. 
+The white light source cannot be turned on until the chiller is running. 
+The chiller must be on for a minimum of 15 minutes after the white light source has been turned off before it can be turned off. 
 
-The chiller must be on for a minimum of 15 minutes after the white light source has been turned off before it can be turned off. The white light source cannot be turned on until the chiller is running. 
+The fluid level of the chiller should be checked occasionally. See the maintenance section below.
 
-The fluid level of the chiller should be checked occasionally. 
+The chiller is controlled via a serial device server (RS-232 to ethernet) with the **ts_ATWhiteLight** CSC
+Information on the Moxa serial device server setup can be found `here <https://ts-electrometer.lsst.io/developer-guide/developer-guide.html#moxa-serial-to-ethernet-converter>`__.
```

Comment:
Doesn't this moxa device using a different setting than the electrometer?
I thought you said it had another baud rate?

Basically we just need the info on how to setup a new one if we swap this out.

Reply:
Yeah, put in the right values.

Reply:
I think you need to reference the section below.
Basically add something like:
"The configuration for the chiller can be found in section `White Light Source Connection`_. 

Or, maybe a better option, just move that content here.